### ISSUE TITLE
Installing compiled Pidgin plugins in Adium?

### ISSUE THREAD

**Issue by Nezteb** [adium/adium#53]
I've tried Googling this, but I've found no clear answers: is it possible to install compiled Pidgin plugins in Adium?

In my particular case, I was able to clone and build both:
- https://github.com/EionRobb/purple-teams
- https://github.com/dylex/slack-libpurple

Since Adium is built on `libpurple`, I assume the plugins are at least partially compatible, but the docs mention `.AdiumPlugin` files; I'm not sure if I can use the shared object files I've compiled? https://adium.im/help/pgs/Miscellaneous-LocationsAdiumFiles.html

Are there docs I am missing, or is this not currently possible?

Any and all information is appreciated! 😄

**Comment by RobFog** [comment]
Hi Noah, I'm not a programmer but as far as I remember libpurple plugins have always required so-called "wrappers". You could have a look at how other libpurple plugins do it, for example https://github.com/tripplet/skypeweb4adium or the ones listed at https://adium.im/help/pgs/Accounts-ListOfServices.html

Edit: I just found there's also https://github.com/victori/slack4adium

The "Writing plugins" section at https://web.archive.org/web/20161110024438/https://trac.adium.im/wiki/Development has some general recommendations for writing plugins.

**Comment by ernstki** [comment]
@Nezteb Yeah, it's odd how dylex/slack-libpurple just off-hand [mentions](https://github.com/dylex/slack-libpurple#legacy-authentication) Adium

> If you're using a front-end (like Adium or Spectrum2) that does not let you set the API token, you can enter your token as the account password instead.

…but doesn't acknowledge that everything's wrapped in a `.framework` on macOS, and I don't think a Mac app would even know what to do with the `libslack.so` that gets copied to your home directory by slack-libpurple's `make install-user`.
